modules/ingester/instance_search_test.go

```diff
@@ -749,3 +750,90 @@ func BenchmarkInstanceSearchUnderLoad(b *testing.B) {
 	// exiting and cleaning up
 	time.Sleep(1 * time.Second)
 }
+
+func TestExtractMatchers(t *testing.T) {
```

Comment:
nit: let's also test a query that does math, so something like `{.http.status_code = 200} = 1+1`